mkosi/__init__.py

```diff
@@ -1596,9 +1596,12 @@ def disable_kernel_install(args: CommandLineArguments, root: str) -> None:
 
 
 def reenable_kernel_install(args: CommandLineArguments, root: str) -> None:
-    if not args.bootable or args.bios_partno is not None or not args.with_unified_kernel_images:
```

Comment:
Maybe replace this with `args.esp_partno is None` so it becomes:

` if not args.bootable or args.esp_partno is None or not args.with_unified_kernel_images:`

Reply:
Ah yes! Thanks I'll test that out